add tags tests

mcgallan opened the pull request:
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

mcgallan committed:
add some tag tests

mcgallan committed:
finished

azure-sdk commented:
**API change check for `Azure.ResourceManager.Resources`**

API changes are not detected in this pull request for `Azure.ResourceManager.Resources`

azure-sdk commented:
**API change check for `Azure.ResourceManager`**

API changes are not detected in this pull request for `Azure.ResourceManager`

mcgallan committed:
delete JitRequest tests

mcgallan committed:
change to playback

Yao725 reviewed (commented):
I found that you have deleted the `JitRequest` tests, is it impossible to create this resource?

mcgallan committed:
subscriptionoperation change

mcgallan committed:
remove addTag()

mcgallan committed:
re-run

mcgallan committed:
record

mcgallan committed:
Merge remote-tracking branch 'upstream/main' into Zihe_AddTagsTests

mcgallan committed:
finished comments

mcgallan committed:
Update RemoveTagAsync.json

mcgallan committed:
record

HarveyLink committed:
Merge remote-tracking branch 'upstream/main' into pr/27913

HarveyLink committed:
Update for Tests

HarveyLink committed:
Remove unnecessary files

HarveyLink committed:
Update RemoveTag()Async.json

HarveyLink committed:
Rerecord

mcgallan commented:
> I found that you have deleted the `JitRequest` tests, is it impossible to create this resource?

There was no jitrequest test in the original test. The class I deleted was created by myself, so I deleted it

mcgallan commented on sdk/resources/Azure.ResourceManager.Resources/tests/Scenario/.editorconfig:
```diff
@@ -0,0 +1,89 @@
+# Rules in this file were initially inferred by Visual Studio IntelliCode from the E:\AzureWork\GitHub\azure-sdk-for-net\sdk\resources\Azure.ResourceManager.Resources\tests\Scenario\ codebase based on best match to current usage at 2022/3/30
```
delete this one
